Directory.Build.props

```diff
@@ -26,5 +25,7 @@
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <PackageProjectUrl>https://github.com/Azure/azure-functions-sql-extension</PackageProjectUrl>
+    <PackageIcon>pkgicon.png</PackageIcon>
```

Comment:
Move the pkgicon.png up to the top level images folder and then update the csproj references to each point to it - we don't need multiple copies of the image in the repo.